OrbitCore/LinuxUtils.cpp

```diff
@@ -96,6 +96,7 @@ void ListModules(pid_t pid,
   struct AddressRange {
     uint64_t start_address;
     uint64_t end_address;
+    bool is_kernel_module;
```

Comment:
Doesn't "kernel module" have a very specific meaning, quite distant from this one? It seems that a simple "is_loadable" would be more appropriate (under the hood this seems to be whether the map is mapped to a file).

Reply:
Right naming is probably not that nice. I was also thinking of something like "is_loadable". However, the "m_FoundPdb" dies something very similar.

Reply:
Either way, the name is incorrect, these maps are not necessarily related to the kernel, they are just maps with a name.

Reply:
Any suggestion? "is_loadable" vs. "can_be_loaded"